docs/level_0/basic_concepts.md

```diff
@@ -0,0 +1,78 @@
+# Basic Concepts
+
+This document describes the basic concepts, artifacts and their relationships of Landscaper. 
+
+## DeployItem
+
+The main objective of Landscaper is to set up large cloud environments with complex relationships consisting of many
+small installation tasks like helm deployments, network setup etc. These elementary installation tasks are defined by 
+DeployItems. There are different types of DeployItems for example for deploying helm charts, plain kubernetes manifests 
+or terraform configurations.
```

Comment:
Shouldn't we have a dedicated "Basic Concepts" Chapter, and not start with "DeployItems" immediately ?

Reply:
What should be the content of such a "Basic Concepts" Chapter? If you go to the [overview](https://github.com/robertgraeff/landscaper/blob/basic-concepts-2/docs/README_new.md) you see a motivation chapter which should contain the even more basic details.